co2eq/countries.json

```diff
@@ -0,0 +1,416 @@
+        "RE":{},
+        "BL":{
+            "HotelFootprintKilogramsPerHotelNight":64.6
+        },
+        "SH":{},
+        "KN":{
+            "HotelFootprintKilogramsPerHotelNight":64.6
+        },
+        "LC":{
+            "HotelFootprintKilogramsPerHotelNight":64.6
+        },
+        "MF":{
+            "HotelFootprintKilogramsPerHotelNight":64.6
+        },
+        "PM":{},
+        "VC":{
+            "HotelFootprintKilogramsPerHotelNight":64.6
+        },
+        "WS":{},
+        "SM":{},
+        "ST":{},
+        "SA":{
+            "HotelFootprintKilogramsPerHotelNight":142.2
+        },
+        "SN":{},
+        "RS":{},
+        "SC":{},
+        "SL":{},
+        "SG":{
+            "HotelFootprintKilogramsPerHotelNight":37.1
+        },
+        "SX":{
+            "HotelFootprintKilogramsPerHotelNight":64.6
+        },
+        "SK":{},
+        "SI":{},
+        "SB":{},
+        "SO":{},
+        "ZA":{
+            "HotelFootprintKilogramsPerHotelNight":40.2
+        },
+        "GS":{},
+        "SS":{},
+        "ES":{
+            "HotelFootprintKilogramsPerHotelNight":18.7
+        },
+        "LK":{},
+        "SD":{},
+        "SR":{},
+        "SJ":{},
+        "SE":{},
+        "CH":{
+            "HotelFootprintKilogramsPerHotelNight":6.9
+        },
+        "SY":{},
+        "TW":{
+            "HotelFootprintKilogramsPerHotelNight":71.1
+        },
+        "TJ":{},
+        "TZ":{},
+        "TH":{
+            "HotelFootprintKilogramsPerHotelNight":54.6
+        },
+        "TL":{},
+        "TG":{},
+        "TK":{},
+        "TO":{},
+        "TT":{
+            "HotelFootprintKilogramsPerHotelNight":64.6
+        },
+        "TN":{},
+        "TR":{
+            "HotelFootprintKilogramsPerHotelNight":51.8
+        },
+        "TM":{},
+        "TC":{
+            "HotelFootprintKilogramsPerHotelNight":64.6
+        },
+        "TV":{},
+        "UG":{},
+        "UA":{},
+        "AE":{
+            "HotelFootprintKilogramsPerHotelNight":94.9
+        },
+        "GB":{
+            "HotelFootprintKilogramsPerHotelNight":17.4,
+            "areas":{
+                "UNITED_KINGDOM":{
+                    "HotelFootprintKilogramsPerHotelNight":17.4
+                },
+                "LONDON":{
+                    "HotelFootprintKilogramsPerHotelNight":20.4
+                }
+            }
+        },
+        "UM":{},
+        "US":{
+            "HotelFootprintKilogramsPerHotelNight":21.4
+        },
+        "UY":{},
+        "UZ":{},
+        "VU":{},
+        "VE":{},
+        "VN":{
+            "HotelFootprintKilogramsPerHotelNight":55.1
+        },
+        "VG":{
+            "HotelFootprintKilogramsPerHotelNight":64.6
+        },
+        "VI":{
+            "HotelFootprintKilogramsPerHotelNight":64.6
+        },
+        "WF":{},
+        "EH":{},
+        "YE":{},
+        "ZM":{},
+        "ZW":{}
+    },
+    "expiryDateHotelFootprintKilogramsPerHotelNight":"2020-07-31"
```

Comment:
What does this represent?

Reply:
The UK government that provided the data considers that they will be outdated by that date. (See this file: https://docs.google.com/spreadsheets/d/1f1j9EeVn9czOZBJKLXvgPwnmldakxuJ7/edit#gid=1584958883 ) I imagine they will provide a new data set when this one expires.
I took example on the file for exchange rates that also has a date key in it to make sure the data are recent enough to be valid. ( https://github.com/tmrowco/tmrowapp-contrib/blob/master/integrations/utils/currency/exchange_rates.json )

Reply:
ok clear